tests/test_decoding.py

```diff
@@ -721,7 +712,7 @@ def test_decode_request_coded_const_undefined_byte_position(self) -> None:
                 "coded_const_parameter_4": 0x56,
             },
         )
-        decoded_message = diag_layer.decode(coded_message)[0]
+        decoded_message = ecu_variant.decode(coded_message)[0]
```

Comment:
please do those renaming changes in a follow up PR

Reply:
ok, I hope I caught the most severe drive-bys in #327 (since everything is intermingeled, the present PR is build on top of it...)